front_end/NavBar.js

```diff
@@ -84,4 +107,5 @@ const stackOption = {
 
 export const tabOption = ["Matching", "Inbox", "Profile"];
 export const BrowseOption = ["BrowseOne", "BrowseTwo", "BrowseThree", "BrowseFour"];
-export const InboxOption = ["InboxOne", "InboxTwo", "InboxThree"];
\ No newline at end of file
+export const InboxOption = ["InboxOne", "InboxTwo", "InboxThree"];
+export const ProfileOption = ["ProfileOne", "Settings"];
```

Comment:
I don't think these should be arrays. I think they should be strings. When they're arrays, then we've changed from calling magic strings to calling magic numbers. 

I agree that it's less lines of code when sorted as an array, but that's not the point here. We can have a separate file of a bunch of strings if needed, but best if there's only one point of maintenance.

Reply:
I agree that we should probably have a file with a bunch of strings, though for a different reason -- when running these, we get a "cycle" warning - basically because NavBar is using the screen files to navigate to them, but these files are also importing the NavBar for these 'option' strings. A way around this is to have a file of constants, which we could potentially use for filenames for this, as well as constants for other displayed strings that we use frequently. (Honestly, it just generally feels wrong to hardcode strings into js files to me.) If we disregard this for this PR, I can add it to my upcoming PR, as long as you guys think this is something we should do.